python/mmSolver/tools/averagemarker/tool.py

```diff
@@ -0,0 +1,97 @@
+"""
+This tool averages markers position from selected markers.
+"""
+
+import maya.cmds
+import mmSolver.tools.selection.filternodes as filternodes
+import mmSolver.logger
+import mmSolver.tools.createmarker.tool
+import maya.mel
+import mmSolver.api as mmapi
+
+LOG = mmSolver.logger.get_logger()
+
+
+def average_marker():
```

Comment:
Please make this function named "main".

Reply:
We should also update the documentation and shelf buttons with this function name change.